config/config.go

```diff
@@ -19,6 +19,8 @@ type Config struct {
 	RobotsFilePath             string        `envconfig:"ROBOTS_FILE_PATH"`
 	KafkaConfig                KafkaConfig
 	OpenSearchConfig           OpenSearchConfig
+	SitemapSaveLocation        string `envconfig:"SITEMAP_SAVE_LOCATION"` // "local" or "s3", default: "local"
```

Comment:
as there are only 2 options, better to use a boolean ?

Reply:
I think it's more explicit this way, the boolean would need to be something like `SITEMAP_S3_ENABLED`, which doesn't tell you what happens when it's disabled.